simphony_ui/tests/test_openfoam_wrapper_creation.py

```diff
@@ -147,3 +166,38 @@ def test_raise(self):
         self.boundary_condition.type = 'coucou'
         with self.assertRaises(ValueError):
             get_boundary_condition_description(self.boundary_condition)
+
+
+class TestOpenfoamMeshCreation(unittest.TestCase):
+
+    def setUp(self):
+        self.openfoam_model = CustomOpenfoamModel()
+        self.openfoam_model.output_path = tempfile.mkdtemp()
+        self.openfoam_model.mesh_name = 'test_mesh'
+        self.openfoam_model.input_file = \
+            os.path.join(
+                os.path.dirname(os.path.dirname(__file__)),
+                'openfoam_input.txt'
+            )
+
+    def test_block_mesh_creation(self):
+        openfoam_wrapper = create_openfoam_wrapper(self.openfoam_model)
+        with cleanup_garbage(self.openfoam_model.output_path):
```

Comment:
I'm not sure... Nothing is done in the setup function, I just set some parameters. That's in the create_openfoam_mesh routine that I try to read an input file and create output files. So that's in this function that I have chances to have failures.

But you're right, that's not what I want, what I want is a tearDown function which will remove all temporary files.

```python
def test_block_mesh_creation(self):
     openfoam_wrapper = create_openfoam_wrapper(self.openfoam_model)
     create_openfoam_mesh(openfoam_wrapper, self.openfoam_model)

def tearDown(self):
    shutil.rmtree(self.openfoam_model.output_path)
```

Am I right ?